src/main/resources/application.properties

```diff
@@ -0,0 +1,2 @@
+quarkus.datasource.db-kind=postgresql
+quarkus.datasource.devservices.volumes."/"=/var/lib/postgresql/

```

Comment:
@l0rd should the "quarkus.datasource.devservices.volume" exist here rather than devfile to use the kubedock and dev services?

How about 3.4.3? Is the latest version fixed as well?

Reply:
It should work with any 3.x version.

Reply:
Setting the volume property via an environment variable isn't supported anymore, so there is no way to specify it via the devfile.

Reply:
Gotcha! Thanks for the clarification.